frontend/src/layout/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -1,16 +1,30 @@
 import './ErrorBoundary.scss'
 
 import { ErrorBoundary as SentryErrorBoundary, getCurrentHub } from '@sentry/react'
-import { useActions } from 'kea'
+import { useActions, useValues } from 'kea'
 import { supportLogic } from 'lib/components/Support/supportLogic'
 import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { teamLogic } from 'scenes/teamLogic'
 
-export function ErrorBoundary({ children }: { children?: React.ReactNode }): JSX.Element {
+export function ErrorBoundary({
+    children,
+    tags = {},
+}: {
+    children?: React.ReactNode
+    tags?: Record<string, any>
```

Comment:
I'd separate the props to an interface and would replace `any` with the `Primitive` [type](https://github.com/getsentry/sentry-javascript/blob/eacf56b0994a6d061452280d7d338502a8e1ff32/packages/types/src/misc.ts#L65) from Sentry. I'm not sure if Sentry serializes objects somehow, but I guess it's not. If someone passes an object, it will most likely be captured as `[Object object]`.